e2e_samples/parking_sensors_synapse/devops/azure-pipelines-cd-release.yml

```diff
@@ -13,7 +13,7 @@ resources:
   - repository: mdwdataops_workspace_publish
     type: github
     endpoint: mdwdops-github
-    name: your_github_handle/your_repo  # deploy script should replace this with your github repo
+    name: devlace/modern-data-warehouse-dataops  # deploy script should replace this with your github repo
```

Comment:
might need to change back to your_github_handle/your_repo

Reply:
ooh good catch

Reply:
Fixed :)